.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   test:
-    runs-on: [self-hosted, linux]
+    runs-on: ubuntu-20.04
     if: contains(github.event.pull_request.labels.*.name, 'PR-approved')
```

Comment:
This line can be removed. It was used to make sure we do not run arbitrary code on our CI machines. As we run on Github runner now, we don't need it anymore.

Reply:
Done